TACC/Core-CMS-Custom
# Task/WP-331: Distinguish Admin versus User submission listings

## Overview
...

## Related

- [WP-331](https://tacc-main.atlassian.net/browse/WP-331)


## Changes
- Changed the title of both View Submissions pages for Users and Admins to reflect accordingly
- For users it is changed to User Submissions
- For admins it is changed to "All Submissions"
- Changed description of "All completed submissions by organizations" to "All submissions by organizations" because the list shows multiple statuses rather than just the completed status submissions

## Testing
- Go to http://localhost:8000/submissions/list-submissions/ and confirm the new title
- Go to http://localhost:8000/administration/list-submissions/ and confirm the new title and description


## UI
Users
![image](https://github.com/user-attachments/assets/015545a1-08f6-4294-901e-079adcca20cc)

Admins
![image](https://github.com/user-attachments/assets/d44089ee-f023-4f81-a750-7f8a1cff339f)
